specification/security/resource-manager/Microsoft.Security/stable/2019-01-01/alerts.json

```diff
@@ -590,7 +590,12 @@
                     "readOnly": true,
                     "type": "string",
                     "description": "Azure resource ID of the workspace that the alert was reported to."
-                }
+                },
+				"correlationKey": {
```

Comment:
minor: could you fix indenting (same comment as in the other spec)?

Reply:
fixed